internal/services/network/application_gateway_resource_test.go

```diff
@@ -7465,6 +7465,11 @@ resource "azurerm_application_gateway" "test" {
     priority                   = 10
   }
 
+  ssl_policy {
+    policy_type = "Predefined"
+    policy_name = "AppGwSslPolicy20150501"
+  }
```

Comment:
Can we update the check for this test instead to ensure the API is returning the documented default value [AppGwSslPolicy20220101](https://learn.microsoft.com/en-us/azure/application-gateway/application-gateway-ssl-policy-overview#predefined-tls-policy) is being returned

https://github.com/hashicorp/terraform-provider-azurerm/blob/c092e2f5bd10cd5c7c41414bb98e86b6595f6c92/internal/services/network/application_gateway_resource_test.go#L1061

```suggestion
```

Reply:
updated